tests/UserTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+use DoSomething\Gateway\Northstar;
+use Rogue\Models\User;
+
+class UserTest extends TestCase
+{
+    /**
+     * Test that an admin user can access a page in the app.
+     *
+     * @return void
+     */
+    public function testAuthenticatedUserDoesntGetRedirectedHome()
+    {
+        $user = factory(User::class)->make([
+            'role' => 'admin',
+        ]);
+
+        $this->actingAs($user)
+            ->visit('/reportbacks')
+            // Only authenticated users will see log out button.
+            ->see('Log Out');
+    }
+
+    /**
+     * Test that non-admin can not get into the site.
+     *
+     * @return void
+     */
+    public function testUnauthenticatedUserCantAccessPagesInApp()
+    {
+        $user = factory(User::class)->make();
+
+        $this->actingAs($user)
+            ->visit('/reportbacks')
+            ->see('You Don\'t have the proper priviledges to do this!');
+    }
+
+    /**
+     * Test that northstar authorization is called when hiting the /login route
+     *
+     * @return void
+     */
+    public function testLogin()
+    {
+        $mock = $this->mock(Northstar::class)
+            ->shouldReceive('authorize')
+            ->andReturn(302);
```

Comment:
I think it's safe to remove the `->andReturn(302);` and just not mock a return value.